examples/templateservicebroker/apiserver-config.yaml

```diff
@@ -0,0 +1,4 @@
+kind: TemplateServiceBrokerConfig
+apiVersion: config.templateservicebroker.openshift.io/v1
+templateNamespaces:
+- openshift
```

Comment:
If we're going to keep this contained within origin, would it make sense to create a role that will ensure this template exists and will apply it (along with all other TSB specific files)?